### DISCUSSION TITLE
自荐一个低代码开源产品O2OA

### DISCUSSION THREAD

**Discussion by xiaotaottt** [taowen/awesome-lowcode#85] [Ideas]
https://www.o2oa.net/
### 平台拥有流程管理、门户管理、信息管理、数据管理和服务管理五大核心能力。用户可以直接使用平台已有功能进行信息信息化建设，平台提供了完整的用户管理，权限管理，流程和信息管理体系，并且提供了大量的开发组件和开箱即用的应用，可以大幅度减化企业信息化建设成本和业务应用开发难度。

**Comment by LowCoderGitHub** [comment]
你好 可以入驻低代码行业交流平台LowCoder平台哦 https://www.lowcoder.com.cn/
